src/vm/methodtablebuilder.cpp

```diff
@@ -7924,7 +7924,8 @@ VOID    MethodTableBuilder::PlaceInstanceFields(MethodTable ** pByValueClassCach
             for (i = 0; i < MAX_LOG2_PRIMITIVE_FIELD_SIZE; i++) {
                 DWORD j;
 
-                if (IS_ALIGNED(dwCumulativeInstanceFieldPos, 1<<(i+1)))
+                size_t alignment = size_t{ 1 } << i;
```

Comment:
There is VS bug that keep showing this warning even when you have an explicit cast to `size_t`:
`size_t alignment = static_cast<size_t>(1 << I); `, the issue was reported.

but there is not harm to declare `1` as `size_t`.